package.json

```diff
@@ -31,21 +31,22 @@
     ]
   },
   "scripts": {
-    "prebuild": "rimraf build/ build-bin/ build-zip/ && patch-package",
-    "build": "npm run build:src && npm run build:bin",
+    "postinstall": "patch-package",
+    "prebuild": "rimraf build/ build-bin/",
+    "build": "npm run build:src",
```

Comment:
* I've renamed `build:bin` to `build:standalone` to make it clearer that what it does is to create the standalone `pkg` bundles (which are later zipped by `npm release` -- hmm thinking about it, it would be even clearer if the zip files were also created by `build:standalone`).
* I've removed `build:bin` (now `build:standalone`) from `npm run build` (which is called by `npm test`). The bottom line is that neither `npm test` nor `npm publish` will create the standalone pkg packages (or the oclif native installers). The motivation was to save build time and make it consistent with native oclif installers.

Am I missing something? I hope not!